aries-site/src/examples/foundation/icons/IconSizeExample.js

```diff
@@ -4,8 +4,11 @@ import { Alarm } from '@hpe-design/icons-grommet';
 
 export const IconSizeExample = () => (
   <Box direction="row-responsive" gap="medium">
+    <Alarm size="xxlarge" />
     <Alarm size="xlarge" />
     <Alarm size="large" />
     <Alarm size="medium" />
+    <Alarm size="small" />
+    <Alarm size="xsmall" />
```

Comment:
The icon legibility gets poor once we get to the small sizes, so recommending addressing partially with color.

However, raises two questions:
- Why do we offer such small sizes? What are the use cases?
- How does one address situations when the use case calls for larger icons?

Followed by:
- How did our icon sizing change so dramatically? What steps did we go through to validate and ensure this is what we wanted?

```suggestion
    <Alarm size="xxlarge" color="icon-strong" />
    <Alarm size="xlarge" color="icon-strong" />
    <Alarm size="large" color="icon-strong" />
    <Alarm size="medium" color="icon-strong" />
    <Alarm size="small" color="icon-strong" />
    <Alarm size="xsmall" color="icon-strong" />
```